psycop/common/feature_generation/sequences/cohort_definer_to_prediction_times.py

```diff
@@ -34,14 +34,23 @@ def _polars_dataframe_to_patient_timestamp_mapping(
         dataframe: pl.DataFrame,
         id_col_name: str,
         patient_timestamp_col_name: str,
+        lookahead: dt.timedelta | None = None,
     ) -> dict[PATIENT_ID, list[dt.datetime]]:
+        max_timestamp: dt.datetime = dataframe[patient_timestamp_col_name].max()  # type: ignore
```

Comment:
Would be even better to get the max timestamp of (outcome_timestamps | prediction_time_timestamps | predictors), but this is much faster, and probably just as good.